src/firejail/checkcfg.c

```diff
@@ -222,6 +223,10 @@ int checkcfg(int val) {
 			else if (strncmp(ptr, "join-timeout ", 13) == 0)
 				join_timeout = strtoul(ptr + 13, NULL, 10) * 1000000; // seconds to microseconds
 
+			// modify default seccomp list
+			else if (strncmp(ptr, "seccomp-list-modify ", 20) == 0)
+				config_seccomp_list_modify = seccomp_check_list(ptr + 20);
```

Comment:
If `seccomp-list-modify` is specified more than once, the last entry wins. It would be better if all entries would be honored. Also the memory is not free()d.

Reply:
Hm, it seems that firejail.config in general doesn't expect settings to be specified more than once currently. I can add that if wanted, but would prefer to collect the changes in a separate pull request.

Reply:
Sorry, I was thinking about the command line switch. Yes, it's indeed material for another PR.